### PR TITLE
doc: typos and grammar fixes on website main page

### PR DESCRIPTION
**Pull request by Bananeweizen**
Fix some typos and grammar on the main web page.

### CODE REVIEW

**Comment by rnveach** [comment]
@Bananeweizen you must build your commit on the latest master.
This is why CI is failing and there is a conflict.

**Comment by Bananeweizen** [comment]
ah thanks, I totally forgot to update my clone first.

File: src/xdocs/index.xml.vm [hunk 1]
```diff
@@ -85,20 +85,20 @@
...
     </section>
 
-    <section name="Development Circle">
+    <section name="Development Cycle">
       <p>
         <b>Releases will happen at the end of each month</b>
-        if functional changes exists in <a href="https://github.com/checkstyle/checkstyle/commits/master">master branch of our repo</a>.
-        For exact day pelase follow our <a href="https://github.com/checkstyle/checkstyle/milestones">milestones</a>.
+        if functional changes exist in the <a href="https://github.com/checkstyle/checkstyle/commits/master">master branch of our repo</a>.
+        For the exact day please follow our <a href="https://github.com/checkstyle/checkstyle/milestones">milestones</a>.
```
Comment (rnveach) [inline src/xdocs/index.xml.vm]: Please add comma after `day`.

Comment (Bananeweizen) [reply src/xdocs/index.xml.vm]: done and rebased on master

**Pull request merged**